website/src/data/team.json

```diff
@@ -190,6 +190,12 @@
       "url": "https://stability.ai/",
       "githubURL": "https://github.com/stability-ai",
       "imageURL": "https://platform.stability.ai/assets/HexagonTextLong.4501db27.png"
+    },
+    "warning": {
+      "name": "Hallucinations",
+      "title": "Although there are safeguards in place, the system may occasionally generate incorrect or misleading information.  It is not intended to give advice.",
+      "url": "https://en.wikipedia.org/wiki/Hallucination_(artificial_intelligence)",
+      "imageURL": "https://assets.stickpng.com/images/5a81af7d9123fa7bcc9b0793.png"
```

Comment:
we probably dont want to add this team section, please move this to inside a component.

Reply:
Will do, thanks for the feedback!